extensions/game.py

```diff
@@ -750,8 +714,7 @@ async def _day_notify_of_night_phase(self):
             if player.executionor_target and not player.executionor_target.dead:
                 player.executionor_target.role = self.ctx.bot.role_mapping["Jester"]()
 
-        # This is where we'll send the day notification
-        # task = self.ctx.create_task()
```

Comment:
Since these comments were removed, can comments on lines 727-728 be removed as well?